content/3_Silver/Intro_Bitwise.mdx

```diff
@@ -342,3 +342,82 @@ def prod(a: int, b: int) -> int:
 </LanguageSection>
 
 <Problems problems="probs" />
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    Which of the following will set the kth bit of int x as true? Assume you do not know what the value of the kth bit currently is.
+    <Quiz.Answer>
+      `x \&= (1 \<\< k)`
+      <Quiz.Explanation>
+        Incorrect. This will set the number to $1<<k$ if the kth bit was true, and 0 if the kth bit was false.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      `x |= (1 \<\< k)`
+      <Quiz.Explanation>
+        Correct. This will set the kth bit to true regardless of what it currently is.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      `x += (1 \<\< k)`
+      <Quiz.Explanation>
+        Incorrect. If the kth bit is already true, it will make the kth bit false and mess up the rest of x.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which gives the number of subsets for an array of size n? (Include the empty subset)
+    <Quiz.Answer>
+      `n \<\< 1`
```

Comment:
just asking, are the backslashes necessary

Reply:
yeah, otherwise it gives error since it thinks '<' is the start of a new section